cmd/soroban-cli/src/print.rs

```diff
@@ -7,6 +7,8 @@ use crate::{
     utils::{explorer_url_for_transaction, transaction_hash},
 };
 
+const TERMS: &[&str] = &["Apple_Terminal", "vscode"];
```

Comment:
How about windows?

Reply:
Didn't test it yet, still need to fix my computer :/

Reply:
Does cmd/powershell even support emojis?

Reply:
powershell does, and that's what we officially support.

Reply:
![CleanShot 2024-09-03 at 16 52 44](https://github.com/user-attachments/assets/07bbee99-3cf0-4d75-b02a-ca25b3be905e)

Reply:
@leighmcculloch @Shaptic just tested `main` on windows terminal and Microsoft got it right! 👏 

![CleanShot 2024-09-04 at 09 51 02](https://github.com/user-attachments/assets/f2d10f89-58d7-4097-9587-a29ac097215d)